NUnitTests/ABS_Pos_Test.cs

```diff
@@ -0,0 +1,24 @@
+using CSharpCalculator;
+
+namespace NUnitTests;
+
+public class ABS_Pos_Test
```

Comment:
No need to add Pos and Neg to test name. Because it is better to move all tests related to one method to one class. And add Tests to class name and Test to method name.

Reply:
I still haven't seen Tests in the class names and Test in the method names.